webserver/views/api/v1/core.py

```diff
@@ -267,6 +267,93 @@ def get_many_highlevel():
     return jsonify(recording_details)
 
 
+def parse_select_features():
+    """Check whether the features are found or not.
+    Parse the query string of features to create a list of 
+    the json paths to features, excluding those that are 
+    not offered.
+
+    If features are missing, an APIBadRequest Exception is
+    raised stating the missing features message.
+    
+    Returns a string of feature paths with alias:
+        "llj.data->'feature1' AS feature1, ..., llj.data->'featureN' AS featureN"
+    """
+    features_param = request.args.get("features")
+    if not features_param:
+        raise webserver.views.api.exceptions.APIBadRequest("Missing `features` parameter")
+
+    selectable_features = ['lowlevel.average_loudness', 
```

Comment:
@alastair I'm unsure of the best place to locate this hardcoded array of features. Maybe it should belong here, or perhaps it should be moved to the bottom of the file at the top level?